### ISSUE TITLE
Create new namespace

### ISSUE THREAD

**Issue by norricorp** [apache/openwhisk#5219]
I have gone through the help and googled with no success.
But how is a new namespace created in openwhisk? The default guest namespace is present when "wsk namespace list" but wsk namespace has two options - get and list. No set or create.
So how is a new namespace created?

**Comment by upgle** [comment]
You can use wskadmin tool (https://github.com/apache/openwhisk/tree/master/tools/admin)

**Comment by norricorp** [comment]
Thanks @upgle. Do I run simply run this as a python script?

**Comment by style95** [comment]
Yes, `wskadmin` would work based on the `whisk.properties` file.

**Comment by ajaysourcedigital** [comment]
is it possible to hit http request to it? or any way we can use any npn package ?

**Comment by style95** [comment]
You can refer to the subject database and create a document in the DB directly.
